conans/client/command.py

```diff
@@ -621,61 +592,49 @@ def imports(self, *args):
 
         try:
             if "@" in args.path and ConanFileReference.loads(args.path):
-                raise ArgumentError(None, "Parameter 'path' cannot be a reference,"
-                                          " but a folder containing a conanfile.py or conanfile.txt"
-                                          " file.")
+                raise ArgumentError(None, "Parameter 'path' cannot be a reference. Use a folder "
+                                          "containing a conanfile.py or conanfile.txt file.")
         except ConanException:
             pass
 
         return self._conan.imports(args.path, args.import_folder, args.install_folder)
 
     def export_pkg(self, *args):
-        """Exports a recipe & creates a package with given files calling 'package'.
-           It executes the package() method applied to the local folders '--source-folder' and
-           '--build-folder' and creates a new package in the local cache for the specified
-           'reference' and for the specified '--settings', '--options' and or '--profile'.
+        """Exports a recipe & creates a package with given files calling 'conan package'. It
```

Comment:
No, it doesn't call ``conan package``, this is confusing. It calls ``package()``